File: web3rpc/rpc-specs/code-samples/python/klay/transaction/getTransactionByHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

transactionHash = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_transaction_by_hash(transactionHash)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const transactionHash = '0x21b2919b89278ca786226f10edbaadced7381dbd73df546a4823547aaebffa58'

    sdk.klay.getTransactionByHash(transactionHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()